addons/spreadsheet/static/src/chart/plugins/odoo_chart_ui_plugin.js

```diff
@@ -86,6 +94,23 @@ export default class OdooChartUIPlugin extends UIPlugin {
                 ) {
                     this._addDomains();
                 }
+
+                const domainEditionCommands = cmd.commands.filter(
+                    (cmd) => cmd.type === "UPDATE_CHART"
+                );
+                for (const cmd of domainEditionCommands) {
+                    if (!(cmd.chartId in this.getters.getOdooChartIds())) {
+                        continue;
+                    }
+
+                    const chartDefinition = this.getters.getChartDefinition(cmd.chartId);
+                    const dataSourceId = this._getOdooChartDataSourceId(cmd.chartId);
+                    this.dataSources.add(dataSourceId, ChartDataSource, chartDefinition);
```

Comment:
why not use `_updateChartDataSourceDefinition` here ?
Shouldn't we check that the domain actually changed, just like above?

Reply:
Yes, you're right, I forgot to update this part to use it here also